### DISCUSSION TITLE
How to map Calculator to M4 key?

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#123] [Q&A] [answered]
Hi,

I want to map the windows calculator to the M4 key. 

So I tried "%windir%\system32\calc.exe" but it doesn't work.

Can you tell me how I should enter the path?

Thanks.

**Comment by seerge** [comment, marked as answer]
Put a full path of exe in a text field ``"C:\Windows\System32\calc.exe"`` and make sure that dropdown near M4 says "Custom"
